modules/gis/src/lib/binary-to-geojson.js

```diff
@@ -1,4 +1,11 @@
-// Top-level function to convert input of binary arrays to GeoJSON
+/**
+ * Convert binary geometry representation to GeoJSON
+ *
+ * @param data   geometry data in binary representation
+ * @param  {string?} type   Input data type: Point, LineString, or Polygon
+ * @param  {string?} format Output format, either geometry or feature
+ * @return GeoJSON objects
+ */
 export function binaryToGeoJson(data, type, format) {
```

Comment:
For exported functions, I recommend keeping JSDoc with the typescript definitions, then it will benefit intellisense in applications.